hugegraph-api/src/main/java/com/baidu/hugegraph/server/RestServer.java

```diff
@@ -65,6 +66,10 @@ public void start() throws IOException {
 
         this.httpServer = this.configHttpServer(uri, rc);
         try {
+            //load for swagger ui
+            this.httpServer.getServerConfiguration().addHttpHandler(
+                    new StaticHttpHandler(
+                    "swagger-ui"), "/swagger-ui");
```

Comment:
could you please improve the code style: https://github.com/apache/incubator-hugegraph-doc/wiki/HugeGraph%E4%BB%A3%E7%A0%81%E9%A3%8E%E6%A0%BC%E6%8C%87%E5%8D%97

Reply:
Forgot it, thanks